uat/testing-features/pom.xml

```diff
@@ -15,7 +15,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <aws.sdk.version>2.17.254</aws.sdk.version>
+        <aws.sdk.version>2.20.157</aws.sdk.version>
```

Comment:
this isn't really used. The version is on LL34

Reply:
whoops, updated